### ISSUE TITLE
Invitations not sent when added meetings from Android 

### ISSUE THREAD

**Issue by biva** [nextcloud/server#9472]
Using Davdroid, no invitation is sent when creating an event with an attendee. It looks very similar to https://github.com/nextcloud/server/issues/1482

The lines added by @ctodobom and @rullzer 
`+$server->addPlugin(new \Sabre\CalDAV\Schedule\Plugin());`
`+$server->addPlugin(new OCA\DAV\CalDAV\Schedule\IMipPlugin( \OC::$server->getMailer(), \OC::$server->getLogger()));`

 are not present in the code: https://github.com/nextcloud/server/blob/master/apps/dav/appinfo/v1/caldav.php

Config:
NC 13.0.2
Davdroid 1.11.2-ose
Calendar 1.6.1

[Edit] : it does work from Thunderbird, invitations are sent

**Comment by biva** [comment]
Thank you @ctodobom . The option is already enabled, and invitation is sent from Thunderbird and the calendar app on the web UI. But not from Android with Davdroid. Any idea?

**Comment by biva** [comment]
Thank you. But no spam :(

How could I investigate what is not working on my side? Tested with the same attendee's email address, both from Thunderbird and the calendar app: the invitation is sent. I don't see anything specific in the log (only the 2 invitations sent from TB and the calendar app).

I tested the creation from 2 apps (acalendar and Google Calendar)

Here is the debug info from Davdroid:

<pre><code>
--- BEGIN DEBUG INFO ---

SOFTWARE INFORMATION
Package: at.bitfire.davdroid
Version: 1.11.2-ose (218) from 5 mai 2018
Installed from: com.android.packageinstaller
JB Workaround installed: no

CONNECTIVITY (at the moment)
Active connection: WiFi, CONNECTED

CONFIGURATION
Power saving disabled: yes
android.permission.READ_CONTACTS permission: granted
android.permission.WRITE_CONTACTS permission: granted
android.permission.READ_CALENDAR permission: granted
android.permission.WRITE_CALENDAR permission: granted
org.dmfs.permission.READ_TASKS permission: denied
org.dmfs.permission.WRITE_TASKS permission: denied
System-wide synchronization: automatically
Account: MyAccount
  Address book sync. interval: 60 min
  Calendar     sync. interval: 60 min
  OpenTasks    sync. interval: —
  WiFi only: false
  [CardDAV] Contact group method: CATEGORIES
  [CalDAV] Time range (past days): 90
           Manage calendar colors: true
Address book account: Perso (MyAccount CA)
  Main account: Account {name=MyAccount, type=bitfire.at.davdroid}
  URL: https://mynextcloud.com/remote.php/dav/addressbooks/users/sisim/Perso/
  Sync automatically: true

SQLITE DUMP
android_metadata
	|  locale |
	|  fr_FR |
----------
services
	|  _id | accountName | service | principal |
	|  3 | MyAccount | carddav | https://mynextcloud.com/remote.php/dav/principals/users/sisim/ |
	|  4 | MyAccount | caldav | https://mynextcloud.com/remote.php/dav/principals/users/sisim/ |
----------
sqlite_sequence
	|  name | seq |
	|  services | 4 |
	|  homesets | 10 |
	|  collections | 23 |
----------
homesets
	|  _id | serviceID | url |
	|  9 | 4 | https://mynextcloud.com/remote.php/dav/calendars/sisim/ |
	|  10 | 3 | https://mynextcloud.com/remote.php/dav/addressbooks/users/sisim/ |
----------
collections
	|  _id | serviceID | type | url | readOnly | forceReadOnly | displayName | description | color | timezone | supportsVEVENT | supportsVTODO | source | sync |
	|  19 | 4 | CALENDAR | https://mynextcloud.com/remote.php/dav/calendars/sisim/contact_birthdays/ | 1 | 0 | Contact birthdays | <null> | -54 | <null> | 1 | 0 | <null> | 0 |
	|  20 | 4 | CALENDAR | https://mynextcloud.com/remote.php/dav/calendars/sisim/personal/ | 0 | 0 | Personal | <null> | -1579148 | <null> | 1 | 1 | <null> | 1 |
	|  21 | 3 | ADDRESS_BOOK | https://mynextcloud.com/remote.php/dav/addressbooks/users/sisim/Perso/ | 0 | 0 | Perso | <null> | <null> | <null> | 0 | 0 | <null> | 1 |
	|  22 | 3 | ADDRESS_BOOK | https://mynextcloud.com/remote.php/dav/addressbooks/users/sisim/Pro/ | 0 | 0 | Pro | <null> | <null> | <null> | 0 | 0 | <null> | 0 |
	|  23 | 3 | ADDRESS_BOOK | https://mynextcloud.com/remote.php/dav/addressbooks/users/sisim/Trash/ | 0 | 0 | Trash | <null> | <null> | <null> | 0 | 0 | <null> | 0 |
----------

SYSTEM INFORMATION
Android version: 7.1.2 (lineage_h815-userdebug 7.1.2 NJH47F c627f1545d)
Device: LGE LG-H815 (h815)

--- END DEBUG INFO ---
</code></pre>

**Comment by ctodobom** [comment]
I suppose that you replaced the address of the "Tricot" creator by Account_Name_On_Davdroid ... Android does this. Can you try to add an event with an external participant through the web interface and see if it sends the message? If it not send, try testing the ability of nextcloud on sending the message from this specific user, maybe you are experiencing an issue with any mail getting out from nextcloud (thunderbird can send it by itself)

**Comment by ctodobom** [comment]
maybe the bug is being triggered by a non-mail user on the atendees. Looks like davdroid always put the current user as an attendee even if it is not a mail address, this may be causing fail when nextcloud try to send the message!

**Comment by rullzer** [comment]
@georgehrke can you look into this?

**Comment by georgehrke** [comment]
@biva Can you please provide server logs? There should be an error message in the web servers error log / Nextcloud log if the invitation is not sent.

**Comment by biva** [comment]
@ctodobom you were right! I should have named the account with my email address, that was the issue. I set a new account with my email address, and it works now. I'm sorry, because I did not read carefully the instruction while creating the account: Davdroid does explain that I should use my email address...

So I proposed a new documentation file, to explain Android sync:
- Android sync: https://github.com/nextcloud/documentation/pull/764 
- Index: https://github.com/nextcloud/documentation/pull/765
